docs/sources/dashboards/previews.md

```diff
@@ -0,0 +1,112 @@
++++
+title = "Dashboard previews"
+keywords = ["grafana", "dashboard", "documentation", "previews"]
+aliases = ["/docs/grafana/latest/reference/previews/"]
+weight = 9
++++
+
+# Previews
+
+{{< figure  max-width="950px" src="/static/img/docs/dashboards/previews.jpg" animated-gif="/static/img/docs/dashboards/previews.gif" >}}
+
+> **Note:** Dashboard previews are available in Grafana 9.0+ as an opt-in beta feature. Please refer to the [permissions]({{< relref "#preview-visibility">}}) section to learn about issues with data source permissions before enabling the feature.
```

Comment:
Should not be a note. Also, move the image to after line 13.
Also, I am not sure about this "to learn about issues with data source permissions before enabling the feature.". What information should the user look for?

Reply:
I like the "note" format since it makes this information "pop" from the doc - it immediately captures reader's attention/ Is there any other way to accomplish this? It's an important limitation and we want everyone reading the doc to read this section first.

Reply:
> Also, I am not sure about this "to learn about issues with data source permissions before enabling the feature.". What information should the user look for?

rephrased